spec/features/user_views_favorite_tattoos_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+feature "User views all favorite tattoos", %(
+As a user
+I want to view a list of my favorite tattoos
+So that I can see all of them at once
+
+  Acceptance Criteria
+  [x] I must be logged in to view my favorites
+  [x] I must be able to access my favorites wherever I am on the site
+  [x] I must be able to view the favorites by clicking the favorites link
+  [x] I must see my favorite tattoos on my favorites show page
+) do
+
+  context 'user views their favorite list' do
+    let!(:user1) { FactoryGirl.create(:user) }
+    let!(:user2) { FactoryGirl.create(:user) }
+    let!(:t1) { FactoryGirl.create(:tattoo, user: user1) }
+    let!(:t2) { FactoryGirl.create(:tattoo, user: user2) }
+    let!(:t3) { FactoryGirl.create(:tattoo) }
+    let!(:favorite1) { FactoryGirl.create(:favorite, user: user2, tattoo: t2) }
+    let!(:favorite2) { FactoryGirl.create(:favorite, user: user1, tattoo: t3) }
+    let!(:favorite3) { FactoryGirl.create(:favorite, user: user1, tattoo: t2) }
+    let!(:favorite4) { FactoryGirl.create(:favorite, user: user2, tattoo: t3) }
```

Comment:
It might be better to use the `user_with_favorites` factory for this! Also, I'm not convinced you need quite *all* of these - maybe trim it down a little?